README.md

```diff
@@ -49,260 +58,85 @@ chmod 700 get_saucectl.sh
-- [TestCafe](https://github.com/saucelabs/sauce-testcafe-runner/releases/latest)
-- [Cypress](https://github.com/saucelabs/sauce-cypress-runner/releases/latest)
-
-
-## Examples
-
-The examples here show how Pipeline testing can be used. Try them and find your own use cases. Every testrunner image comes with a preconfigured setup that allows you to focus on writing tests instead of tweaking with the configurations. Our initial testrunner flavors come either with Puppeteer or Playwright as an automation framework. They will start the browser for you and expose the `browser` object ([Puppeteer](https://pptr.dev/#?product=Puppeteer&version=v3.0.3&show=api-class-browser) / [Playwright](https://playwright.dev/#version=v1.0.1&path=docs%2Fcore-concepts.md&q=browser)) to the global scope for you to be accessible in the test:
-
-#### Puppeteer Snippet:
-
-Our Puppeteer testrunner image exposes `browser` into the global scope which represents an instance of its [`Browser class`](https://pptr.dev/#?product=Puppeteer&version=v3.0.4&show=api-class-browser). The browser will be initiated and shutdown by the testrunner setup.
-
-```js
-describe('saucectl demo test', () => {
-	test('should verify title of the page', async () => {
-		const page = (await browser.pages())[0]
-		await page.goto('https://www.saucedemo.com/');
-		expect(await page.title()).toBe('Swag Labs');
-	});
-});
-```
-
-#### Playwright Snippet:
-
-The Playwright testrunner image also exposes a global `browser` variable that represents Playwright's [`Browser class`](https://playwright.dev/#version=v1.0.2&path=docs%2Fcore-concepts.md&q=browser). In addition to that you also have access to a pre-generated [browser context](https://playwright.dev/#version=v1.0.2&path=docs%2Fcore-concepts.md&q=browser-contexts) via `context` as well as to a [page frame](https://playwright.dev/#version=v1.0.2&path=docs%2Fcore-concepts.md&q=pages-and-frames) via `page`.
-
-```js
-describe('saucectl demo test', () => {
-	test('should verify title of the page', async () => {
-		await page.goto('https://www.saucedemo.com/');
-		expect(await page.title()).toBe('Swag Labs');
-	});
-});
-```
-
-#### Testcafe Snippet
-```js
-import { Selector } from 'testcafe';
-fixture `Getting Started`
-	.page `http://devexpress.github.io/testcafe/example`
-
-const testName = 'testcafe test'
-test(testName, async t => {
-	await t
-		.typeText('#developer-name', 'devx')
-		.click('#submit-button')
-		.expect(Selector('#article-header').innerText).eql('Thank you, devx!');
-});
-```
-
-#### Cypress Snippet
-```js
-context('Actions', () => {
-	beforeEach(() => {
-		cy.visit('https://example.cypress.io/commands/actions')
-	})
-	it('.type() - type into a DOM element', () => {
-		// https://on.cypress.io/type
-		cy.get('.action-email')
-			.type('fake@email.com').should('have.value', 'fake@email.com')
-	})
-})
+specify it through:
+ 
+```shell script
+saucectl run -c ./path/to/config.yml
```

Comment:
@spider-sauce in regards to the docs do we want to show our examples with the long version name of the flag?
```
saucectl run -c ./path/to/config.yml
```
vs
```
saucectl run --config ./path/to/config.yml
```